hera_qm/ant_metrics.py

```diff
@@ -0,0 +1,394 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import aipy as a 
+from copy import deepcopy
+from pyuvdata import UVData
+import json
+
+
+#######################################################################
+#Low level functionality that is potentially reusable
+#######################################################################
+
+
+def per_antenna_modified_z_scores(metric):
+    '''For a given metric, stored as a (ant,antpol) dictonary, computes the per-pol modified z-score 
```

Comment:
Ok, I agree simple ones are nice. I think this one's right on the cusp because it's where you define the metric dictionary. I'll leave it up to you.